server/seeders/index.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const basename = path.basename(module.filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(`${__dirname}/../config/config.json`)[env];
+const db = {};
+
```

Comment:
How come you have two of this file ?band why is this in your seeders folder. Do you intend on merging this too ?